clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -88,6 +87,7 @@ public class CommitRequestManager implements RequestManager, MemberStateListener
     private final boolean throwOnFetchStableOffsetUnsupported;
     final PendingRequests pendingRequests;
     private boolean closing = false;
+    private Map<TopicPartition, OffsetAndMetadata> allConsumed = Map.of();
```

Comment:
I find the name a little confusing. (I know we get it from `SubscriptionState`.) What about `latestPartitionOffsets` or something? If nothing else, please add a comment to make it clear what this map contains.

This might be a stylistic choice, but what's the advantage of this over using `Collections.emptyMap()`?